src/input/command.zig

```diff
@@ -0,0 +1,407 @@
+                .action = .{ .new_split = .down },
+                .title = "Split Down",
+                .description = "Split the terminal down.",
+            },
+        },
+
+        .toggle_split_zoom => comptime &.{.{
+            .action = .toggle_split_zoom,
+            .title = "Toggle Split Zoom",
+            .description = "Toggle the zoom state of the current split.",
+        }},
+
+        .equalize_splits => comptime &.{.{
+            .action = .equalize_splits,
+            .title = "Equalize Splits",
+            .description = "Equalize the size of all splits.",
+        }},
+
+        .reset_window_size => comptime &.{.{
+            .action = .reset_window_size,
+            .title = "Reset Window Size",
+            .description = "Reset the window size to the default.",
+        }},
+
+        .inspector => comptime &.{.{
+            .action = .{ .inspector = .toggle },
+            .title = "Toggle Inspector",
+            .description = "Toggle the inspector.",
+        }},
+
+        .open_config => comptime &.{.{
+            .action = .open_config,
+            .title = "Open Config",
+            .description = "Open the config file.",
+        }},
+
+        .reload_config => comptime &.{.{
+            .action = .reload_config,
+            .title = "Reload Config",
+            .description = "Reload the config file.",
+        }},
+
+        .close_surface => comptime &.{.{
+            .action = .close_surface,
+            .title = "Close Terminal",
+            .description = "Close the current terminal.",
+        }},
+
+        .close_tab => comptime &.{.{
+            .action = .close_tab,
+            .title = "Close Tab",
+            .description = "Close the current tab.",
+        }},
+
+        .close_window => comptime &.{.{
+            .action = .close_window,
+            .title = "Close Window",
+            .description = "Close the current window.",
+        }},
+
+        .close_all_windows => comptime &.{.{
+            .action = .close_all_windows,
+            .title = "Close All Windows",
+            .description = "Close all windows.",
+        }},
+
+        .toggle_maximize => comptime if (!builtin.os.tag.isDarwin()) &.{.{
+            .action = .toggle_maximize,
+            .title = "Toggle Maximize",
+            .description = "Toggle the maximized state of the current window.",
+        }} else &.{},
+
+        .toggle_fullscreen => comptime &.{.{
+            .action = .toggle_fullscreen,
+            .title = "Toggle Fullscreen",
+            .description = "Toggle the fullscreen state of the current window.",
+        }},
+
+        .toggle_window_decorations => comptime if (!builtin.os.tag.isDarwin()) &.{.{
+            .action = .toggle_window_decorations,
+            .title = "Toggle Window Decorations",
+            .description = "Toggle the window decorations.",
+        }} else &.{},
+
+        .toggle_secure_input => comptime if (builtin.os.tag.isDarwin()) &.{.{
```

Comment:
Actually, secure input is already implemented on Linux (#6054). For the future IMO we should leave the responsibility of filtering out unimplemented features to the apprt, since core shouldn't really be aware of what's implemented/relevant and what's not

Reply:
Agree on the filtering. 

But secure input is not implemented in Linux that’s a different feature. Secure input is a macOS only system call that prevents all event taps so keyloggers aren’t possible. Ghostty automatically enables secure input on macOS when it detects a password prompt or users can do it manually.